kratos/utilities/embedded_skin_utility.cpp

```diff
@@ -277,10 +335,87 @@ namespace Kratos
             case GeometryData::KratosGeometryType::Kratos_Tetrahedra3D4:
                 return Kratos::make_shared<DivideTetrahedra3D4>(rGeometry, rNodalDistances);
             default:
-                KRATOS_ERROR << "Asking for a non-implemented modified shape functions geometry.";
+                KRATOS_ERROR << "Asking for a non-implemented divide geometry utility.";
         }
     }
 
+    template<std::size_t TDim>
+    ModifiedShapeFunctions::UniquePointer EmbeddedSkinUtility<TDim>::SetModifiedShapeFunctionsPointer(
```

Comment:
create?

Reply:
Renamed to `pCreateModifiedShapeFunctions`.